src/main/java/io/candydoc/ddd/tactical_pattern/Repository.java

```diff
@@ -0,0 +1,24 @@
+package io.candydoc.ddd.tactical_pattern;
+
+import io.candydoc.ddd.model.DDDPattern;
+import java.util.Set;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+@SuperBuilder
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class Repository extends TacticalModel {
+  private static final Set<DDDPattern> ALLOWED_DEPENDENCIES = Set.of(TacticalPattern.AGGREGATE);
```

Comment:
A repository can also depend on domain entities.

The repository should have its own PR.